src/GitHub.hs

```diff
@@ -7,7 +7,7 @@
 -- this package.
 --
 -- See "GitHub.Request" module for executing 'Request', or other modules
--- of this package (e.g. "GitHub.Users") for already composed versions.
+-- of this package (e.g. "GitHub.Data.Users") for already composed versions.
```

Comment:
this is incorrect. (some) "composed" requests are in `Endpoints` modules, in this case e.g. http://hackage.haskell.org/package/github-0.21/docs/GitHub-Endpoints-Users.html

Reply:
My mistake. Fixed.